packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -195,6 +202,15 @@ abstract class SliverChildDelegate {
   }
 }
 
+/// A Key that is used in [SliverChildBuilderDelegate] and [SliverChildListDelegate]
+/// that provide the wrapping logic same as ValueKey<Key>.
```

Comment:
`same as [ValueKey<Key>]`

Reply:
not sure what this sentence means though

Reply:
removed